extension/src/popup/views/AccountHistory.tsx

```diff
@@ -176,18 +179,32 @@ const HistoryItem = ({
   );
 };
 
-export const AccountHistory = ({
-  publicKey,
-  operations,
-}: {
-  publicKey: string;
-  operations: Array<HorizonOperation>;
-}) => {
-  const { isTestnet } = useSelector(settingsNetworkDetailsSelector);
+const defaultAccountHistory = {
+  operations: [],
+} as AccountHistoryInterface;
+
+export const AccountHistory = () => {
+  const publicKey = useSelector(publicKeySelector);
+  const networkDetails = useSelector(settingsNetworkDetailsSelector);
+  const [accountHistory, setAccountHistory] = useState(defaultAccountHistory);
 
   const STELLAR_EXPERT_URL = `https://stellar.expert/explorer/${
-    isTestnet ? "testnet" : "public"
+    networkDetails.isTestnet ? "testnet" : "public"
   }`;
+  const { operations } = accountHistory;
+
+  useEffect(() => {
+    const fetchAccountHistory = async () => {
+      try {
+        const res = await getAccountHistory({ publicKey, networkDetails });
+        setAccountHistory(res);
+      } catch (e) {
+        console.error(e);
+      }
+    };
+    fetchAccountHistory();
+  }, [publicKey, networkDetails]);
+
```

Comment:
this used to get Account History passed to it from Account, but now it just fetches it on its own. This was needed to make the `/account-history` route work